lib/routes/llamaindex/blog.ts

```diff
@@ -0,0 +1,97 @@
+import { type Data, type DataItem, type Route, ViewType } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import { type Context } from 'hono';
+
+export const handler = async (ctx: Context): Promise<Data> => {
+    const limit: number = Number.parseInt(ctx.req.query('limit') ?? '15', 10);
+
+    const baseUrl = 'https://www.llamaindex.ai';
+    const targetUrl = `${baseUrl}/blog`;
+
+    const response = await ofetch(targetUrl);
+    const $ = load(response);
+    const language = $('html').attr('lang') ?? 'en';
+
+    const links = new Set<string>();
+    $('a').each((_, element) => {
```

Comment:
Use CSS class name and id to narrow the selection instead of selecting all elements

Reply:
Thanks for the review! Fixed in https://github.com/DIYgod/RSSHub/pull/20328/commits/e380aee48cfa81340f76cbab48fdbbf3271a943f